pkgs/by-name/we/weblate/package.nix

```diff
@@ -26,12 +26,10 @@ let
 in
 python.pkgs.buildPythonApplication rec {
   pname = "weblate";
-  version = "5.8.3";
+  version = "5.9.2";
 
   pyproject = true;
 
-  disabled = python.pythonOlder "3.11";
```

Comment:
Why remove this? Upstream says the same thing https://github.com/WeblateOrg/weblate/blob/main/pyproject.toml#L167C1-L167C27